src/Symfony/Bundle/FrameworkBundle/Resources/config/debug_prod.xml

```diff
@@ -5,7 +5,7 @@
     xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
 
     <parameters>
-        <parameter key="debug.error_handler.throw_at">0</parameter>
+        <parameter key="debug.error_handler.throw_at">-1</parameter>
```

Comment:
Why do we have a parameter for that, we've tried to remove them for 3.0, so not sure why we kept this one specifically. Or was it an oversight?